README.md

```diff
@@ -159,6 +159,7 @@ Device notes are sorted alphabetically, major (upper case) notes before minor
 | Power supply       | [Corsair RM650i, RM750i, RM850i, RM1000i](docs/corsair-hxi-rmi-psu-guide.md) | <sup>_h_</sup> | 1.12.1 |
 | Power supply       | [NZXT E500, E650, E850](docs/nzxt-e-series-psu-guide.md) | <sup>_p_</sup> | 1.7.2 |
 | LED controller     | [Aquacomputer Farbwerk 360](docs/aquacomputer-farbwerk360-guide.md) | <sup>_hp_</sup> | 1.11.1 |
+| LED controller     | [Aquacomputer Farbwerk](docs/aquacomputer-farbwerk-guide.md) | <sup>_hp_</sup> | 1.13.1 |
```

Comment:
If the device is so similar to the Farbwerk 360, does it really need a separate entry?

Also, the minimum recommended version would be `git`, not (a future/fictitious) `1.13.1`.